interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -680,14 +680,15 @@ function postcalendar_admin_categories($msg = '', $e = '', $args = array())
     $output->SetOutputMode(_PNH_RETURNOUTPUT);
     $output->SetOutputMode(_PNH_KEEPOUTPUT);
 
-    $form_hidden = "<input type=\"hidden\" name=\"is_update\" value=\"" . attr($is_update) . "\" />";
-    $form_hidden .= "<input type=\"hidden\" name=\"pc_event_id\" value=\"" . attr($pc_event_id) . "\" />";
+    $form_hidden = "<input type=\"hidden\" name=\"is_update\" value=\"" . attr($is_update ?? '') . "\" />";
+    $form_hidden .= "<input type=\"hidden\" name=\"pc_event_id\" value=\"" . attr($pc_event_id ?? '') . "\" />";
     if (isset($data_loaded)) {
         $form_hidden .= "<input type=\"hidden\" name=\"data_loaded\" value=\"" . attr($data_loaded) . "\" />";
-        $tpl->assign('FormHidden', $form_hidden);
     }
+    $tpl->assign('FormHidden', $form_hidden);
```

Comment:
not sure about this here, might have to add an else that blanks the `value` attribute maybe

Reply:
Odd since looks like $form_hidden elements not added unless isset($data_loaded). Maybe something like:
```
    if (isset($data_loaded)) {
        $form_hidden = "<input type=\"hidden\" name=\"is_update\" value=\"" . attr($is_update ?? '') . "\" />";
        $form_hidden .= "<input type=\"hidden\" name=\"pc_event_id\" value=\"" . attr($pc_event_id ?? '') . "\" />";
        $form_hidden .= "<input type=\"hidden\" name=\"data_loaded\" value=\"" . attr($data_loaded ?? '') . "\" />";
        $tpl->assign('FormHidden', $form_hidden);
    }
```
And then in smarty if it is outputting FormHidden, only do it there if FormHidden is not empty